api/src/main/java/javax/mvc/MvcContext.java

```diff
@@ -123,4 +125,12 @@
      * @return instance of encoders.
      */
     Encoders getEncoders();
+
+    /**
+     * Returns the locale of the current request.
+     *
+     * @return The request locale
+     */
```

Comment:
I'd keep it in a single line to avoid ambiguity in docs. I.e:
```
/**
 * @return the locale of the current request.
*/
```
What do you think?